`Uint8Array.prototype.toString` does not take any argument

```
packages/pocketjs/transaction-builder/models/msgs/msg-proto-app-stake.ts:83:62 - error TS2554: Expected 0 arguments, but got 1.

83          value: MsgProtoStake.encode(data).finish().toString('base64'),
                                                                ~~~~~~~~

packages/pocketjs/transaction-builder/models/msgs/msg-proto-app-unjail.ts:47:58 - error TS2554: Expected 0 arguments, but got 1.

47          value: MsgUnjail.encode(data).finish().toString('base64'),
                                                            ~~~~~~~~

packages/pocketjs/transaction-builder/models/msgs/msg-proto-app-unstake.ts:46:64 - error TS2554: Expected 0 arguments, but got 1.

46          value: MsgBeginUnstake.encode(data).finish().toString('base64'),
                                                                  ~~~~~~~~

packages/pocketjs/transaction-builder/models/msgs/msg-proto-node-stake.ts:95:66 - error TS2554: Expected 0 arguments, but got 1.

95          value: MsgProtoNodeStake.encode(data).finish().toString('base64'),
                                                                    ~~~~~~~~

```

Solution: Use `Buffer`
```javascript
Buffer.from(Uint8ArrayData).toString('base64')
```

--- a/packages/transaction-builder/src/models/msgs/msg-proto-app-stake.ts
+++ b/packages/transaction-builder/src/models/msgs/msg-proto-app-stake.ts
@@ -32,7 +32,9 @@ export class MsgProtoAppStake extends TxMsg {
     if (isNaN(amountNumber)) {
       throw new Error('Amount is not a valid number')
     } else if (amountNumber < MINIMUM_STAKE_AMOUNT) {
-      throw new Error('Amount should be bigger than ${MINIMUM_STAKE_AMOUNT} uPOKT')
+      throw new Error(
+        'Amount should be bigger than ${MINIMUM_STAKE_AMOUNT} uPOKT'
+      )
     } else if (this.chains.length === 0) {
       throw new Error('Chains is empty')
     }
@@ -80,7 +82,9 @@ export class MsgProtoAppStake extends TxMsg {
 
     return Any.fromJSON({
       typeUrl: this.KEY,
-      value: MsgProtoStake.encode(data).finish().toString('base64'),
+      value: Buffer.from(MsgProtoStake.encode(data).finish()).toString(
+        'base64'
+      ),
     })
   }
 }

--- a/packages/transaction-builder/src/models/msgs/msg-proto-app-unjail.ts
+++ b/packages/transaction-builder/src/models/msgs/msg-proto-app-unjail.ts
@@ -44,7 +44,7 @@ export class MsgProtoAppUnjail extends TxMsg {
 
     return Any.fromJSON({
       typeUrl: this.KEY,
-      value: MsgUnjail.encode(data).finish().toString('base64'),
+      value: Buffer.from(MsgUnjail.encode(data).finish()).toString('base64'),
     })
   }
 }

--- a/packages/transaction-builder/src/models/msgs/msg-proto-app-unstake.ts
+++ b/packages/transaction-builder/src/models/msgs/msg-proto-app-unstake.ts
@@ -43,7 +43,9 @@ export class MsgProtoAppUnstake extends TxMsg {
 
     return Any.fromJSON({
       typeUrl: this.KEY,
-      value: MsgBeginUnstake.encode(data).finish().toString('base64'),
+      value: Buffer.from(MsgBeginUnstake.encode(data).finish()).toString(
+        'base64'
+      ),
     })
   }
 }

--- a/packages/transaction-builder/src/models/msgs/msg-proto-node-stake.ts
+++ b/packages/transaction-builder/src/models/msgs/msg-proto-node-stake.ts
@@ -103,7 +103,9 @@ export class MsgProtoNodeStakeTx extends TxMsg {
 
     return Any.fromJSON({
       typeUrl: this.KEY,
-      value: MsgProtoNodeStake.encode(data).finish().toString('base64'),
+      value: Buffer.from(MsgProtoNodeStake.encode(data).finish()).toString(
+        'base64'
+      ),
     })
   }
 }

